### ISSUE TITLE
"Sync up" language is confusing

### ISSUE THREAD

**Issue by BigBlueHat** [w3c/dpub-pwp-ucr#163]
I know you just merged #162, but the "sync up" language is confusing (especially given the vast usage of the word "sync"--especially in offline related apps/things).

Maybe "Cross-format identifiable reading locations" comes closer?

**Comment by atyposh** [comment]
Hi Benjamin,

I should have read the text more closely. The merge request came from @iherman and I was just about to begin some general editorial reorganization of the entire document, so I just merged it in.

Upon closer inspection, this entire section seems to be about two different features, IMO. The first feature is locating content in a WP based on its location in the print. The second is tracking (and syncing) progress through a WP across devices with varying viewports.

The first two examples of 2.1.12 (this new section) are very similar to the fourth example of 2.1.10 (Pagination). While print page numbers (for myriad reasons) may not be available to a WP (or may never have existed), this is a common request that should be supported where possible. The last example in 2.1.12 is clearly about tracking progress from one device to another, but especially from one viewport to another.

So, I think I may merge this content into my ongoing rewrite of the "Pagination" section (which itself will be getting a less ambiguous label). I'd like to clearly distinguish between jumping to a print page number (not always possible) and tracking progress across devices (always possible, unless the reader is anonymous or otherwise opts out of progress tracking).

Thoughts?

**Comment by BigBlueHat** [comment]
At the core I think there's a common "position identifier"--(print-provided) page numbers and viewport location being a couple of examples.

How those identifiers are moved between devices, user agents, or people is beyond scope (I'd reckon), but would require that a position/location be expressible at the very least.

So conceptually, I think they're still a single unit.

**Comment by atyposh** [comment]
Reading progress (syncing between devices of varying viewports) cannot be determined by the print page number. First of all, I have never seen XML that says "begin page 56" and "end page 56". Post-print XML will have page ranges in the head, but only if it was printed. WPs cannot depend upon something having been printed. Even with page ranges for a chapter, for example, there is no way to know that page 400 on a phone (which would have been page 40 in print) is page 80 on a tablet. Put another way, page 40 in print could be 10 different pages on a phone (or more if there are images).

Am I missing something?

**Comment by BigBlueHat** [comment]
It's all about affordances. 😉 

WPs certainly should not *depend* on something being printed, but *if* something has been printed (i.e. now has references assigned to regions of content), then a WP should afford the ability to express that (perhaps: `/moby-dick/#p2`).

In the case of view port position (irrespective of other available identifiers), the screen dimensions can't (easily) be re-usable among different view port sizes, so some other "positioning"/locating system would be better (perhaps: [Web Annotations](https://www.w3.org/TR/annotation-model/)).

Regardless (to me at least), the goal seem definable as wanting to provide "cross-format identifiable reading locations"--be they author/publisher provided (page numbers, HTML anchors) or user/reader generate-able (Web Annotation, SVG's [#svgView()](https://www.w3.org/Graphics/SVG/Test/20011026/linking-view-BE-04.svg#svgView(viewBox(279,276,152,104))) fragment identifiers).

All those seem to boil down to having a consistent way to express content-related positioning regardless of devices shape.

**Comment by llemeurfr** [comment]
IMHO we should avoid stratospheric terms like "Cross-format identifiable reading locations" and stay near practical terms. 

A "pointer as identified in a physical book" is a page number, isn't it? let's call it page number then. 
(note for French people: vous rappelez-vous du référentiel bondissant ;-) ? ).

A bookmark is a different beast, as it is not "cross-format" in the physical vs digital sense. But we'd like bookmarks to be precise enough so that people can find the bookmarked information whatever their UA is; this is a "cross-screen" persistence. On a physical book we bookmark a page (2 pages in fact). In a digital book we should be able to bookmark a pointer to a character or set of characters. 

Therefore I suggest to split the use case in:
- Ability to indicate a physical page number
- Ability to bookmark a precise position in the publication

The examples can be those already present in the UCR document.

**Comment by BigBlueHat** [comment]
> * Ability to indicate a physical page number
> * Ability to bookmark a precise position in the publication

When imagining a Web Publication "encoding" of a physical book, how are those two things different?

**Comment by TzviyaSiegman** [comment]
> Ability to indicate a physical page number
Ability to bookmark a precise position in the publication

When imagining a Web Publication "encoding" of a physical book, how are those two things different?

Being able to align with print page numbering is important when there is a print version
Being able to bookmark a precise position is always important.

**Comment by BigBlueHat** [comment]
And isn't a "page number" expressed within a digital edition (of a previously physical book) simply a "precise position" which someone may or may not choose to "bookmark"?

**Comment by francofaa** [comment]
Hi everyone,

I have a pull request related to this issue ( #189 ). I have left comments in the comments of the pull request for people to have input into. 

_Ability to indicate a physical page number:_ Some of the use cases for the original page number requirement did not relate to page numbers. I have moved them since they are more related to bookmarking. I have also changed the language for this requirement and moved it to Random Access to Content since Synchronization did not seem to be an accurate reflection of what it was. Synchronization implies something is automatically happening whereas this page numbering requirement and use cases described a more manual cross referencing.

_Ability to bookmark a precise position in the publication:_ This requirement already existed, but it has been rephrased to be more explicitly related to bookmarking. Two of the use cases from page numbering have been moved here. The Aika use case is less clear and I would like input on this one.

I have also added another requirement for preserving reading state, which is related to this topic.
